.gitignore

```diff
@@ -35,3 +35,6 @@ skyrim_inventory_management.csr
 /vendor/bundle/
 
 .pgdata
+
+# Ignore CSV files exported from canonical data (or imported to generate canonical data)
+/lib/tasks/canonical_models/*.csv
```

Comment:
In general these should not be committed, especially since they can always be generated again with a single Rake task.